tooling/create-tauri-app/src/recipes/svelte.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019-2021 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+import { join } from 'path'
+import { shell } from '../shell'
+import { Recipe } from '../types/recipe'
+
+const completeLogMsg = `
+  Your installation completed.
+  To start, run \`yarn dev\` and \`yarn tauri dev\`
```

Comment:
Should probably tell users to `yarn install` or `npm install` first then `yarn tauri dev` or `npm run tauri -- dev` 
something like this 

```js
console.log(`
    Your installation completed.
    
    $ cd ${cfg.appName}
    $ ${packageManager === 'yarn' ? 'yarn' : 'npm run'} tauri ${
      packageManager === 'npm' ? '--' : ''
    } dev
  `)
```

Reply:
I tried a fresh install, and it doesn't seem like yarn/npm install is needed, and `npm run tauri dev` works without the `--`. Should I still add them as suggestions?

Reply:
if `yarn/npm install` isn't need, don't add them. As for `--`, add them because I think some version of npm only works with them

Reply:
Good to know. Ok, I added it.